### ISSUE TITLE
Can we send email from Android (K9-Mail) to PC(Microsoft Outlook)?

### ISSUE THREAD

**Issue by norani** [open-keychain/open-keychain#1821]
I want to know whether smartphone(Android using K9-mail) can send/receive email  (also can encrypt and decrypt by using Openkeychain) to/from PC (PC use Microsoft Outlook for encrypt and decrypt) ?

I tried send sign from phone to PC but PC cannot detect the signature.. The recipient(PC) only can view like below, not detect it as signed email 
-----BEGIN PGP SIGNATURE-----
XXXXXXXXXXX(cipher)
-----END PGPG SIGNATURE------

**Comment by pbeckmann** [comment]
Hello,

that output you shown is the correct output. The not detected signature comes from a missing pgp plugin, or integration. It's not a bug in my opinion, just a misconfiguration. 

Are you sure that you configured outlook correctly with pgp?

Greetings from Germany

**Comment by norani** [comment]
After make some research and work, I realized that Outlook block .cer file format. In order to send public key from phone to PC is by open OpenKeyChain>>choose key>>share via>>K9-Mail.

Because Outlook support S/MIME not OpenPGP, another program needed to decrypt/encrypt data. So I used gpg4win(PGP Program) in PC for decrypt and encrypt. As a conclusion, PC and Android can communicate through the helped by OpenKeyChain and gpg4win for encrypt and decryption.
